lib/generators/rolemodel/webpacker/webpacker_generator.rb

```diff
@@ -57,6 +57,11 @@ def add_jest_config
       copy_file 'FilePathMock.js', 'spec/javascript/support/FilePathMock.js'
     end
 
+    def add_rspec_helper
+      say 'Add RSpec helper'
+      copy_file 'spec/support/webpacker.rb'
```

Comment:
I wasn't 100% sure if this helper should be in the Webpacker generator or the Testing generator, since it's related to testing when Webpacker is in use. Thoughts?

Reply:
Webpacker is added by default. So I think this actually makes sense to be in the testing generator